bundles/plugins/org.bonitasoft.studio.form.preview/src/org/bonitasoft/studio/form/preview/FormPreviewOperation.java

```diff
@@ -284,8 +284,9 @@ protected void undeployDeployedProcess(final AbstractProcess process, final Proc
     }
 
     protected void undeployProcess(final ProcessAPI processApi, final ProcessDeploymentInfo info) throws ProcessDefinitionNotFoundException, DeletionException {
-        if (info.getName().equals(lastProcessDeployed) && info.getVersion().equals(formPreviewInit.VERSION)) {
+        if (info.getName().equals(lastProcessDeployed) && info.getVersion().equals(AbstractFormPreviewInitialization.VERSION)) {
             try {
+                processApi.deleteProcessInstances(info.getProcessId(), 0, 10);
```

Comment:
sure that this line i snot more related to BS-14269 https://bonitasoft.atlassian.net/browse/BS-14269